### ISSUE TITLE
Need to remove mapfolder prefix from MapName in database

### ISSUE THREAD

**Issue by broxen** [Segs/Segs#499]
### Issue Summary

Create a new character. Log out and Shutdown Server.
Open Database, character map in entity_data includes map prefix from settings.cfg.

This creates two issues. 
- One, if the map prefix is changed, the user is defaulted to Outbreak (because their map is not found)
- Two, I'm getting errors like this multiple times at the charsel screen. 
```
Warning : No matching map name in g_defined_map_datas to sent map name. Returning Outbreak's display map name as default...
```

### Proposed Fix
Need to remove prefix from MapName, and add it when needed for map_path.

**Comment by eos428** [comment]
For No.2, it seems that character select screen wants to find the map data for all 8 characters in the slots, even if the character is not yet created.

I think this issue is reliant on the results of #494 - if we use MapName, then this issue is prevalent. If we use MapIdx to get map data, then this won't be necessary.

**Comment by broxen** [comment]
Sounds right to me. Good call

**Comment by broxen** [comment]
Closed by #503
